README.md

```diff
@@ -251,6 +252,7 @@ ddev examples
 ddev format
 ddev gen-test-runner
 ddev saucelabs
+ddev task-runner
```

Comment:
Given the description above it might actually be nice to name the command something to do with CI. Liked `run-ci` or something, to make it clear that this isn't meant to be used interactively.